GitUI/BranchTreePanel/RepoObjectsTree.cs

```diff
@@ -491,6 +492,58 @@ List<TreeNode> SearchTree(string text, IEnumerable<TreeNode> nodes)
                     }
                 }
 
+                // if matches with high priority are found bring those to the first of the list by rotating the list
+                // Rotation is done to maintain the tree order, so that on click of search button multiple times,
+                // the selected node passes down the tree to other matches.
+                if (matchIndex > 0)
+                {
+                    rotateListToSetFirstAsMatchIndex(ret, matchIndex);
+                }
+
+                // A function to see if the node text or full path of a node (for branch node) is matching with search text
+                bool searchForMatchingText(string nodeText, string searchText, ref bool isFullMatch, ref int matchIndex, List<TreeNode> ret)
```

Comment:
```suggestion
                static bool searchForMatchingText(string nodeText, string searchText, ref bool isFullMatch, ref int matchIndex, List<TreeNode> ret)
```

Reply:
1. Incorrect casing, all members except for fields must be in PascalCase.
2. Methods' names that return `bool` must start with `Is`, `Has`, `Does`, etc. verb.
```suggestion
                static bool HasMatchingText(string nodeText, string searchText, ref bool isFullMatch, ref int matchIndex, List<TreeNode> ret)
```